web/gds-user-ui/src/components/Section/IntegrateAndComply.tsx

```diff
@@ -61,124 +60,168 @@ export default function IntegrateAndComply() {
-                    <Trans
-                      id="Since TRISA is an open source, peer-to-peer Travel Rule solution, VASPs can set
-                    up and maintain their own TRISA server to exhange encrypted Travel Rule
-                    compliance data. TRISA maintains an">
-                      Since TRISA is an open source, peer-to-peer Travel Rule solution, VASPs can
-                      set up and maintain their own TRISA server to exhange encrypted Travel Rule
-                      compliance data. TRISA maintains an
-                    </Trans>{' '}
-                    <Link href="https://github.com/trisacrypto/trisa" isExternal color="link">
-                      <Trans id="GitHub repository">GitHub repository</Trans>
-                    </Link>{' '}
-                    <Trans
-                      id="with detailed documentation, a reference implemenation, and “robot” VASPs for
-                    testing purposes.">
-                      with detailed documentation, a reference implemenation, and “robot” VASPs for
-                      testing purposes.
-                    </Trans>
-                  </Line>
-                  <Line title={t`Option 2. Use a 3rd-party Solution`}>
-                    <Trans
-                      id="TRISA is designed to be interoperable. There are several Travel Rule solutions
-                    providers available on the market. If you are a customer, work with them to
-                    integrate TRISA into your Travel Rule compliance workflow.">
-                      TRISA is designed to be interoperable. There are several Travel Rule solutions
-                      providers available on the market. If you are a customer, work with them to
-                      integrate TRISA into your Travel Rule compliance workflow.
-                    </Trans>
-                  </Line>
-                  <Line title={t`How to set up your own node?`} fontStyle="italic">
-                    <Trans
-                      id="Talk to a member of your technical team to determine the requirements and
-                    resources to integrate TRISA with your system. Have members of your technical
-                    team integrate your systems with TRISA. Or work with a solutions provider that
-                    can help your VASP set up your TRISA server and maintain it.">
-                      Talk to a member of your technical team to determine the requirements and
-                      resources to integrate TRISA with your system. Have members of your technical
-                      team integrate your systems with TRISA. Or work with a solutions provider that
-                      can help your VASP set up your TRISA server and maintain it.
-                    </Trans>
-                  </Line>
-                  <Line title={t`3rd Party Travel Rule Providers`}>
-                    <UnorderedList color={'#1F4CED'}>
+              <Box pt={4}>
```

Comment:
This differs from the Figma design as the table component extended outside the container even with a max width set. I attempted a grid with 3 columns, but then ran into an issue with some of the text inside a grid item aligning in the center although a text align style had not been applied.